config/webpack.config.prod.js

```diff
@@ -400,6 +393,11 @@ module.exports = {
               'sass-loader'
             ),
           },
+          // SVGR is an tool that converts your SVG files into React components that you can use directly in JXS.
+          {
```

Comment:
currently react-native is setup with react-native-rewrite and doesnt use these webpack scripts, can you check if we can change the webpack config there?

Reply:
@sirpy metro bundler is used and yes, we can reconfigure webpack in metro.config.js.
@yaroslav-fedyshyn-nordwhale already did it

Reply:
for web react-app-rewired is used, it allows to reconfigure webpack too, in the config-overrides.js

Reply:
yes, I figured it out and moved the configs to the right place (config-overrides.js and metro.config.js)